generator/src/routes/+page.svelte

```diff
@@ -61,33 +140,68 @@
       </div>
     </form>
   </div>
   <!-- Right Panel -->
   <div class="qrcode-area-wrapper">
     <div class="qrcode-area">
-      <div class="qrcode">
-        <img class="frame" src={SubtractSvg} alt="" aria-hidden="true" />
+      <div class="qrcode-frame">
+        {#if status !== Status.GENERATING}
+          <img class="frame" src={SubtractSvg} alt="" aria-hidden="true" />
+          <img class="logo" src={LogoTagsSvg} alt="" aria-hidden="true" />
+          {#if status === Status.FINISHED}
+            <img class="qrcode" alt="" aria-hidden="true" src="" bind:this={qrcodeElement} />
+            <div class="ioxio-tag-frame">
+              <img class="tag" src={LogoTagsBlackSvg} alt="" aria-hidden="true" />
+            </div>
+          {/if}
+        {:else}
+          <div class="frame anim" />
+        {/if}
         <img class="effect" src={EffectSvg} alt="" aria-hidden="true" />
-        <img class="logo" src={LogoTagsSvg} alt="" aria-hidden="true" />
       </div>
     </div>
-    <div class="status">IOXIO Tag generator</div>
+    <div class="status">
+      {#if status === Status.READY}
+        IOXIO Tag generator
+      {:else if status === Status.GENERATING}
+        Generating...
+      {:else}
+        Finished!
+      {/if}
+    </div>
     <div class="product-area">
-      <div class="description">
-        <p>
-          This application serves as a demonstration platform for generating QR codes specifically
-          designed for Digital Product Passports.
-        </p>
-        <p>
-          Utilizing the IOXIO Dataspace technology, this app showcases how QR codes can be generated
-          to serve as digital passports for various Data Products. These QR codes can be read using
-          a compatible reader application, enabling access to relevant product details in real time.
-        </p>
-        <a class="documentation" href="/">See documentation →</a>
-      </div>
+      {#if status === Status.READY}
+        <div class="description">
+          <p>
+            This application serves as a demonstration platform for generating QR codes specifically
+            designed for Digital Product Passports.
+          </p>
+          <p>
+            Utilizing the IOXIO Dataspace technology, this app showcases how QR codes can be
+            generated to serve as digital passports for various Data Products. These QR codes can be
+            read using a compatible reader application, enabling access to relevant product details
+            in real time.
+          </p>
+          <a class="documentation" href="/">See documentation →</a>
+        </div>
+      {:else if status === Status.FINISHED}
+        <div class="result">
+          <p class="label">Issuer domain</p>
+          <p class="value">{inputData.iss}</p>
+          <p class="label">Issuer domain</p>
+          <p class="value">{inputData.product}</p>
+          <p class="label">Issuer domain</p>
+          <p class="value">{inputData.id}</p>
```

Comment:
![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/1bb53aa2-e03d-4e13-81f1-c677744d0644)

![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/e1c44035-457f-481d-9a60-b1a1fc20e6e1)

Reply:
So you resolved the `Issuer domain` being repeated 3x exactly how?

Reply:
What do you mean?

Reply:
![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/5a34041b-297a-40de-a31e-8c7ff58964ce)